### ISSUE TITLE
search issue

### ISSUE THREAD

**Issue by ghost** [amzrk2/hugo-theme-fuji#83]
![image](https://user-images.githubusercontent.com/17074394/123008365-0cade480-d3bb-11eb-8efd-4fb69907e359.png)
![image](https://user-images.githubusercontent.com/17074394/123008565-61515f80-d3bb-11eb-9b8e-c223054c9219.png)

search seems to show duplicate results (and also is not very accurate)
is it possible to add the option to use some other search method?
for example:
https://gist.github.com/eddiewebb/735feb48f50f0ddd65ae5606a1cb41ae

**Comment by JoinGitHub1** [comment]
I agree. Search does not work. You can type jibberish in the search bar and still get hits. I don't understand how this works but hoping it could be fixed.
